gui/src/test/java/io/bisq/gui/util/validation/AltCoinAddressValidatorTest.java

```diff
@@ -568,4 +568,18 @@ public void testREF() {
         assertFalse(validator.validate("2a65Aca4D5fC5B5C859090a6c34d16413539822g").isValid);
         assertFalse(validator.validate("").isValid);
     }
+    
+    @Test
+    public void testSTL() {
+        AltCoinAddressValidator validator = new AltCoinAddressValidator();    
+        validator.setCurrencyCode("STL");
+        
+        assertTrue(validator.validate("Se3x7sVdvUnMMn2KoYLyYVHMJGRoB2R3V8K3LYuHAiEXgVac7vsmFiXUC8dSpJnjXDfwytKsQJV6HFH8MjwPagTJ2Aha46RZM").isValid);
+        assertTrue(validator.validate("Se3F51UzpbVVnQRx2VNbcjfBoQJfeuyFF353i1jLnCZda9yVN3vy8csbYCESBvf38TFkchH1C1tMY6XHkC8L678K2vLsVZVMU").isValid);
+        
+        assertFalse(validator.validate("Se3x7svUnMMn2KoYLyYVHMJGRoB2R3V8K3LYuHAiEXgVac7vsmFiXUC8dSpJnjXDfwytKsQJV6HFH8MjwPagTJ2Aha46RZM").isValid);
```

Comment:
This test does not tell me why is this input invalid. It would be good to assign that string to a variable with a meaningful name or at least add a comment.

Reply:
I'll add a comment right away